exporter.go

```diff
@@ -49,7 +49,7 @@ type prometheusVersion struct {
 }
 
 const (
-	defaultCollectors            = "cpu,cs,logical_disk,net,os,service,system,textfile"
+	defaultCollectors            = "cpu,cs,logical_disk,physical_disk,net,os,service,system,textfile"
```

Comment:
Does it make sense to have both the `logical_disk` and `physical_disk` collectors enabled by default? That is, do the metrics exposed by both collectors overlap in scope?